dafni_cli/models/upload.py

```diff
@@ -35,6 +35,10 @@ def upload_model(
         json (bool): Whether to print the raw json returned by the DAFNI API
     """
 
+    if not is_valid_definition_file(definition_path):
+        click.echo("Wrong definition file type.")
```

Comment:
Could we also print out the valid file types, perhaps something like
`Wrong definition file type '.zip', valid file types are '.json', '.yml' or '.yaml'`